Symfony/CS/Tests/Fixer/AbstractFixerTestBase.php

```diff
@@ -37,6 +37,30 @@ protected function getTestFile($filename = __FILE__)
         return $files[$filename];
     }
 
+    /**
+     * Tests if a fixer fixes a given string to match the expected result.
+     *
+     * It is used both if you want to test if something is fixed or if it
+     * is not touched by the fixer. It also makes sure that the expected
+     * output does not change when run through the fixer. That means that
+     * you do not need two test cases like
+     * [$expected] and
```

Comment:
I would just not keep to that ugly 70 limit.
Especially when the following code is wider.